src/components/Header/Header.jsx

```diff
@@ -40,10 +40,12 @@ const navbarPages = [
 const burgerItems = () => [
   {
     component: <AccountWidget key={0} />,
+    key: 0,
     close: true,
   },
-  ...navbarPages.map(item => ({
+  ...navbarPages.map((item, index) => ({
     component: item,
+    key: item.props.to || index,
```

Comment:
we should avoid using index here?

Also if item.props is undefined item.props.to will crash the application

Reply:
check the entire structure:
item && item.props && items.props.to

Although we'll have to put some unique key value here if any of those are undefined/null

Reply:
perhaps we can use `item.key` ?

Reply:
oh yeah that's probably better